### DISCUSSION TITLE
Clio RS EDC, I ruined my radio

### DISCUSSION THREAD

**Discussion by Andres86s** [cedricp/ddt4all#1154] [Q&A]
Hi all, I bought a thinkdiag obd and wrongly ruined/deleted configurations on my radio, now i have only a screen with the message “unconfigured radio” I bought ddt4all and completely realized that I’m not up to the job, can someone please help?
![2580AB71-EDF0-41A9-A5B5-DF20126278E3](https://user-images.githubusercontent.com/84362172/118579783-5a985280-b754-11eb-9d3b-12b4c0e9448c.jpeg)

**Comment by testeur70** [comment]
You bought Ddt4all? The soft is free how did you manage to buy a free soft...

**Comment by AndyIreland** [comment]
you could try this see if it works: 

Fill out the date (as todays date - the date you programme it) 

Car maker = RS sport (but even Renault will be OK)

touch screen = present

MSM = fitted

![rlink recovery 1](https://user-images.githubusercontent.com/49172217/119048030-929ed000-b9b6-11eb-959c-8d9a073b8d71.jpg)

then fill out the VIN and CRC with your details 

then press the home button 5 times - or remove radio fuse inside car leave for 10 mins and put back in and see if its enough filled out to get the radio  working again?

**Comment by Andres86s** [comment]
> You bought Ddt4all? The soft is free how did you manage to buy a free soft...

Sorry, perhaps used incorrect word, I obtained ddt4all after the beer donation.

**Comment by Andres86s** [comment]
> you could try this see if it works:
> 
> Fill out the date (as todays date - the date you programme it)
> 
> Car maker = RS sport (but even Renault will be OK)
> 
> touch screen = present
> 
> MSM = fitted
> 
> ![rlink recovery 1](https://user-images.githubusercontent.com/49172217/119048030-929ed000-b9b6-11eb-959c-8d9a073b8d71.jpg)
> 
> then fill out the VIN and CRC with your details
> 
> then press the home button 5 times - or remove radio fuse inside car leave for 10 mins and put back in and see if its enough filled out to get the radio working again?

Thanks so much Andy...unfortunately I think I just made it worse. Now I also have the frightening “add eol settings”. Any recommendations?

**Comment by AndyIreland** [comment]
OK - try this , format a 8gb SD Card to fat32 and copy the attached tomtom.000 onto the SD card. then put it into the SD reader of the car - then press the home button 5 times to reboot the R-LINK radio and see if it fixes it. If it does you may have to put in a security code to get the R-Link radio unlocked , you can get the code from your owners manual, or the 'my renault' app or from renault themselves if you give them your VIN number - good luck! 

https://gofile.io/d/zuOTmd
(Just click 'I HAVE A VPN ALREADY' - if asked)

**Comment by Andres86s** [comment]
> OK - try this , format a 8gb SD Card to fat32 and copy the attached tomtom.000 onto the SD card. then put it into the SD reader of the car - then press the home button 5 times to reboot the R-LINK radio and see if it fixes it. If it does you may have to put in a security code to get the R-Link radio unlocked , you can get the code from your owners manual, or the 'my renault' app or from renault themselves if you give them your VIN number - good luck!
> 
> https://gofile.io/d/zuOTmd
> (Just click 'I HAVE A VPN ALREADY' - if asked)

Thanks Andy, I just did what you've recommended but unfortunately nothing changed. Any other idea or should I pressume I messed up for good?

**Comment by AndyIreland** [comment]
> > OK - try this , format a 8gb SD Card to fat32 and copy the attached tomtom.000 onto the SD card. then put it into the SD reader of the car - then press the home button 5 times to reboot the R-LINK radio and see if it fixes it. If it does you may have to put in a security code to get the R-Link radio unlocked , you can get the code from your owners manual, or the 'my renault' app or from renault themselves if you give them your VIN number - good luck!
> > https://gofile.io/d/zuOTmd
> > (Just click 'I HAVE A VPN ALREADY' - if asked)
> 
> Thanks Andy, I just did what you've recommended but unfortunately nothing changed. Any other idea or should I pressume I messed up for good?

Thats a shame i was really hoping that would help you because I got that Add EOL screen in my Renault Zoe and on restart with the SD card with that TomTom.001 file on the SD card and filling in the VIN and CRC and a few other settings on restart it just asked me to input the security code for the R-LINK and when I did that it all came to life again. 

What about disconnecting the battery and leaving it disconnected for over 10 minutes and reconnecting it? - I suppose you have already tried that? - if you have i am at a loss - your Clio could have a BIC module in it which my Zoe hasnt so maybe when you wiped the configurations maybe it wiped some important detail in the BIC? - its just a guess though at this stage. - if its not that then maybe Renault will have to have it in , take apart the MFD or radio part and find test pins inside on the circuit board and flash the firmware back in that way to it?

**Comment by Andres86s** [comment]
> > > OK - try this , format a 8gb SD Card to fat32 and copy the attached tomtom.000 onto the SD card. then put it into the SD reader of the car - then press the home button 5 times to reboot the R-LINK radio and see if it fixes it. If it does you may have to put in a security code to get the R-Link radio unlocked , you can get the code from your owners manual, or the 'my renault' app or from renault themselves if you give them your VIN number - good luck!
> > > https://gofile.io/d/zuOTmd
> > > (Just click 'I HAVE A VPN ALREADY' - if asked)
> > 
> > 
> > Thanks Andy, I just did what you've recommended but unfortunately nothing changed. Any other idea or should I pressume I messed up for good?
> 
> Thats a shame i was really hoping that would help you because I got that Add EOL screen in my Renault Zoe and on restart with the SD card with that TomTom.001 file on the SD card and filling in the VIN and CRC and a few other settings on restart it just asked me to input the security code for the R-LINK and when I did that it all came to life again.
> 
> What about disconnecting the battery and leaving it disconnected for over 10 minutes and reconnecting it? - I suppose you have already tried that? - if you have i am at a loss - your Clio could have a BIC module in it which my Zoe hasnt so maybe when you wiped the configurations maybe it wiped some important detail in the BIC? - its just a guess though at this stage. - if its not that then maybe Renault will have to have it in , take apart the MFD or radio part and find test pins inside on the circuit board and flash the firmware back in that way to it?

Thanks as always Andy. I actually took it to the Renault dealer and they say that the unit has a short circuit that doesn’t allow them to configure it. Does thar sound reasonable to you? You have to know that in Mexico, dealers are know to be worse than crap so perhaps they are trying to fool me?

**Comment by AndyIreland** [comment]
> Thanks as always Andy. I actually took it to the Renault dealer and they say that the unit has a short circuit that doesn’t allow them to configure it. Does thar sound reasonable to you? You have to know that in Mexico, dealers are know to be worse than crap so perhaps they are trying to fool me?

i couldnt say for certain sorry, but i have not heard or read of a short circuit - but I have read somewhere , and i am pretty sure it was the Rlink (not 100% because I read articles on the renault medianav as well) but anyway this article I read said if radio cannot be recovered in other ways they have had to take radio apart and solder a wire to the main board or eeprom and rescue it that way by loading firmware .. I will have a search and see if I can find that article again for you , give me a few days

**Comment by Furtif** [comment]
https://github.com/cedricp/ddt4all/issues/836

**Comment by bigmac5753** [comment]
where do I find the CRC?

**Comment by AndyIreland** [comment]
there is a 'CRC calculator' bult into DDT4ALL on the top menu bar - just put in your car's VIN number and it will give you the CRC

**Reply by Yaya38** [reply]
> there is a 'CRC calculator' bult into DDT4ALL on the top menu bar - just put in your car's VIN number and it will give you the CRC

hello, im facing the same problem, uncofigured radio, i did what u sait here i have The CRC, but where should i put it please ?

**Comment by bigmac5753** [comment]
thanks.

done that and entered them into the boxes but still has same error, unconfigured radio.

**Reply by Yaya38** [reply]
hello, where did u put the CRC ?

**Reply by AndyIreland** [reply]
if rlink1 system put crc here for the radio 2012:
![image](https://github.com/user-attachments/assets/06f80ce5-19e4-4dc6-a89d-253ade2001cf)

and then in the navigation settings here:
![image](https://github.com/user-attachments/assets/306abfae-b880-46de-a9ff-d390964f161f)

**Reply by Yaya38** [reply]
Thank you

**Comment by AndyIreland** [comment]
> thanks.
> 
> done that and entered them into the boxes but still has same error, unconfigured radio.

takes a lot more than just writing CRC into box to get unconfigured radio working again.
What did you originally do, press the soft reset button did you?

**Comment by bigmac5753** [comment]
yes.

I did everything above and at first it didn't seem to work but i've just gone and tried again and it worked but it had a speaker test tone that I couldn't turn off.  I tried resetting (home x 5) and it now asks for a pin code.

**Reply by AlMeu** [reply]
> yes.
> 
> I did everything above and at first it didn't seem to work but i've just gone and tried again and it worked but it had a speaker test tone that I couldn't turn off. I tried resetting (home x 5) and it now asks for a pin code.


At these units, if it is R-Link 1, there are not pin code.

**Comment by AndyIreland** [comment]
> yes.
> 
> I did everything above and at first it didn't seem to work but i've just gone and tried again and it worked but it had a speaker test tone that I couldn't turn off. I tried resetting (home x 5) and it now asks for a pin code.

oh well you are nearly there now then - all you have to do is get your radio code from the my renault website after you register your car and add your VIN number - then the test code will be gone

**Comment by bigmac5753** [comment]
I have a smart 453 which is basically a twingo 3.  I assume I can't do this through renault?  can't find anywhere on smart to do it.

**Comment by AndyIreland** [comment]
> I have a smart 453 which is basically a twingo 3. I assume I can't do this through renault? can't find anywhere on smart to do it.

ah right - I dont know where you would go for that radio code then if that is the car you have but you could look here it might give you pointers:
https://www.evilution.co.uk/?s=rlink
https://www.evilution.co.uk/mod/453-r-link-radio-decoder.htm?highlight=radio%20code

https://play.google.com/store/apps/details?id=com.nigmatech.radiocodegeneratorforrenault&hl=en_IE&pli=1

**Reply by morganclio** [reply]
@AndyIreland i think the other page has gone and can't reply.  I've see it that you replied somewhere to download this file to sd card...
I bought this cable as I didn't know that you need the splitter to reconfig the radio. I thought that is only needed for android activation https://www.ebay.co.uk/itm/274067691348?_skw=elm327&itmmeta=01JQ4AABNP6F87760G9RZRNB2S&hash=item3fcfb51754:g:~bYAAOSwxKFdsycQ&itmprp=enc%3AAQAKAAAA4FkggFvd1GGDu0w3yXCmi1fZJmpfz5dGxR9HbXthxZTh5ckY%2F%2BoNuRy98%2FPRS9YCv7k3tHdVTobilD55HmV0q4ynJcZ8Gz51sF3XkPX9%2BgMmerlEl84PH8aOv1uMfQaR0FyDBDrjIA7RQDD0KFjqjcFdh0MFCFuzNbvTUJj05qbVf9DuISr1B%2F46B1vxNZSvQXJHhLcGWYaNePlciluaphv9BATEUzJ1N7Vk%2FZf9mp3DraG0qxUV3ACllcWTIQSPY8tICUgQfwsZceV%2BgLYZ%2BnaSntQQbNfv%2BrEbhltJgmNb%7Ctkp%3ABk9SR_y6qYq5ZQ

**Reply by AndyIreland** [reply]
yes i see that , for some reason wordpress has suspended my account today/this afternoon with no reason , just said i broke some terms and conditions so i am just waiting back for contact from them as to what terms & conditions I have broke!!

you can give this file a try if you like see if it works , it has fixed rlink1 EOL message with some and with others it had no effect , then you have to fill in all missing parameters using DDT4ALL / PYREN - and yes to read and write parameters to your rlink1 through the car's obd2 socket you have to either adapt the wiring of your ELM327 or adapt the wiring on a obd2 fly cable otherwise you cannot work on the rlink1 through the obd2 socket.

here is the file if you want to give it a go. - you have to format a SD card (not your map sd card!) to fat32 and put the TOMTOM.000 onto the root of the SD card, leave the engine running dont turn it off because if engine off the radio may shutdown in the middle of loading the software. 
https://workupload.com/file/sEEYcpnryaW

**Reply by morganclio** [reply]
Ok so I will wait for the ELM327 to turn up, I will check the file and order the splitter cable and swap the cable places.

**Reply by morganclio** [reply]
@AndyIreland so I got 2GB card,formated  to fat32 and moved the tomtom.000 file to SD card.  Put in the slot ,start the car and nothing. Not loading,instaling etc. Just tomtom logo for few seconds and back to the Yellow Triangle.
Am I doing something wrong ,or my card reader is not reading the card?

**Reply by AndyIreland** [reply]
> @AndyIreland so I got 2GB card,formated to fat32 and moved the tomtom.000 file to SD card. Put in the slot ,start the car and nothing. Not loading,instaling etc. Just tomtom logo for few seconds and back to the Yellow Triangle. Am I doing something wrong ,or my card reader is not reading the card?

Try (with the card in the slot) pressing the 'home' button on the rlink1 next to the touchscreen and see if it starts loading - give it a good 20minutes just in case something is going on in the background even though yellow triangle showing onscreen . 

If that no good get a 8gb or more USB stick and put that TOMTOM.000 onto root of usb stick formatted to FAT32 and try loading it that way (again, press home button 5 times to reboot rlink1) - if still yellow triangle after that , there could be a whole load of parameters missin in the radio ECU and MFD ECU and you will have to fill them in usin the ELM327 and cable you are waiting on.

**Reply by AndyIreland** [reply]
> @AndyIreland so I got 2GB card,formated to fat32 

By the way - try a 8GB SD Card , I know the TomTom.000 is only a small file but still try a 8GB SD card , thats what I used to recover my rlink once  - besides the SD card in the Zoe might not recognise a 2GB SD card size it might be too small (jut like a 32GB or 64GB SD card might be too big to be recognised by the SD card reader in the Zoe.

**Comment by morganclio** [comment]
This is what I got now on the screen...

**Reply by AndyIreland** [reply]
did you post a picture , because i cannot see it?

**Reply by morganclio** [reply]
![20250325_224732](https://github.com/user-attachments/assets/03f98580-2c74-40ff-a529-ea49cfef3d66)

**Reply by AndyIreland** [reply]
ok thanks - when you get your elm327 and cable all it might need now is the paeameters like VIN number and CSC and all the other parameters filling out and you might be lucky it might come to life

**Reply by Furtif** [reply]
Try last release. 🤨

**Reply by morganclio** [reply]
> Try last release. 🤨

Where from ?

**Reply by morganclio** [reply]
> ok thanks - when you get your elm327 and cable all it might need now is the paeameters like VIN number and CSC and all the other parameters filling out and you might be lucky it might come to life

@AndyIreland  the screen now has gone back to the yellow triangle. I need to get usb stick and try it that way. Had few at home but when needed can't find them...

**Reply by Furtif** [reply]
> > Try last release. 🤨
> 
> Where from ?

but you don't follow the current events? I bet you're using 2007-12 stuff with a 20xx car

https://github.com/cedricp/ddt4all/releases

**Reply by AndyIreland** [reply]
> > ok thanks - when you get your elm327 and cable all it might need now is the paeameters like VIN number and CSC and all the other parameters filling out and you might be lucky it might come to life
> 
> @AndyIreland the screen now has gone back to the yellow triangle. I need to get usb stick and try it that way. Had few at home but when needed can't find them...

yep always the way - here is another TomTom.000 fix you could give a try if you like by usb or SD card way and see if that fixes it:
https://we.tl/t-Is6XU3Cdqa

Also good if you are gonna try the USB way is to try USB's with a LED light in them , so at least if nothing on screen and the light is flashing/flickering in the USB it will sort of show you that it is reading from the USB stick - take out the original Map SD Card if going down the USB installation way just in case the SD card is out of date.

**Reply by Furtif** [reply]
If the radio is bad, the TomTom won't solve it, the same thing happens with the MediaNav, if the ECU is bad, the cards won't solve it

**Reply by Furtif** [reply]
The radio works without the maps option, this is an option. It erased or formatted the ECU or the radio and asks to configure the ECU. The map option, whether medianav or another, is an option and is not available in certain cars and this can be activated by micomtest in certain devices. Now, if you did something wrong, such as “make virgin” etc., you must reconfigure the ECU or go to the garage or install a radio other than the original or pluginplay.

**Reply by morganclio** [reply]
> If the radio is bad, the TomTom won't solve it, the same thing happens with the MediaNav, if the ECU is bad, the cards won't solve it

@Furtif It was all working fine until I decided to connect Launchx431... I was going through some options but don't remember to click the "reset" (no reason for) and the unit lost the configuration. I did writ the VIN using launch but it didn't solve anything and now have the yellow triangle with "unconifigured radio" message. I put the sd card with the TomTom000 yesterday and no difference. Today I use different card with same file and I did received the  "gotoyourlocaldealer" message.  I stop the car, close is and after a while the screen gone black. Open the car and it went back to the yellow triangle screen. So i do believe that now the radio need to be configured again to work?

**Reply by Furtif** [reply]
I'm talking about the first image in this post, it's a medianav 1 and is IGo not tomtom is version 4.x.x base winCe

**Reply by Furtif** [reply]
> > If the radio is bad, the TomTom won't solve it, the same thing happens with the MediaNav, if the ECU is bad, the cards won't solve it
> 
> @Furtif It was all working fine until I decided to connect Launchx431... I was going through some options but don't remember to click the "reset" (no reason for) and the unit lost the configuration. I did writ the VIN using launch but it didn't solve anything and now have the yellow triangle with "unconifigured radio" message. I put the sd card with the TomTom000 yesterday and no difference. Today I use different card with same file and I did received the "gotoyourlocaldealer" message. I stop the car, close is and after a while the screen gone black. Open the car and it went back to the yellow triangle screen. So i do believe that now the radio need to be configured again to work?

and strange.. the radio in the photo above is old and a medianav 1 or 2 came after firmware 4.x.x to 9.x.x and all in Windows Ce and equipped with IGO and not tomtom, they do not have CarPlay or android auto only later in medianav 3 with Linux instead of winCe which has this option with usb wire in medianav 3 version and/or usb wire and Wi-Fi in medianav 4 version

**Reply by morganclio** [reply]
> I'm talking about the first image in this post, it's a medianav 1 and is IGo not tomtom is version 4.x.x base winCe

My have tomtom navi. The first photo of this post is not my photo

**Reply by Furtif** [reply]
> > I'm talking about the first image in this post, it's a medianav 1 and is IGo not tomtom is version 4.x.x base winCe
> 
> My have tomtom navi. The first photo of this post is not my photo

Navi is IGo not tomtom

**Reply by Furtif** [reply]
Usb update of is NaviSync is IGO

**Reply by Furtif** [reply]
Go to group Facebook https://m.facebook.com/groups/140606960090892/ ask Jeremy Dacia boy solve you problem

**Reply by Furtif** [reply]
Navigo https://github.com/Furtif/MediaNavIGO média nav

**Reply by Furtif** [reply]
**provider maps MediaNav 1 to 3** is HERE IGo

**Reply by Furtif** [reply]
Now you have a old mediaNav Put maps in usb and enter in winCe for update maps (crack mode) or uses toolbox official

**Reply by Furtif** [reply]
Version 4.x.x base Windows Ce micomtest codes here https://cdn.discordapp.com/attachments/1194646634060062763/1311061802083418132/MediaNav.pdf?ex=67e45f73&is=67e30df3&hm=57a538da391afc25563a8bf08802c1c9ab33bc857e7dc47f616238b66b31bbb3&

**Comment by AndyIreland** [comment]
@morganclio if you try the ELM327 and modified cable way , download PyREN, select CAN2 and make sure it reads / can find both ECU's . The rlink 1 uses 2 ECU's and they should show up in PyREN. one should say something like Radio 2012 (which is radio in metal case up behind dashboard somewhere)  and the other one should say MFD (thats the touchscreen) or navigation. 
If say Radio 2012 is missing I think for ICE cars (petrol and diesel) the BIC ECU has to be modified in a Script Box with a script number for your car , something like that , to 'wake it up' - my car where I rescued it is a Zoe EV and it does not have the BIC ecu but ICE Renaults do apparently. There was a guy who gave instruction about wiring modification for the BIC and what script number to put in to wake up the ECU (either Radio2012 or MFD) sorry its very confusing but I will see if I can find it.

**Reply by Furtif** [reply]
> @morganclio if you try the ELM327 and modified cable way , download PyREN, select CAN2 and make sure it reads / can find both ECU's . The rlink 1 uses 2 ECU's and they should show up in PyREN. one should say something like Radio 2012 (which is radio in metal case up behind dashboard somewhere) and the other one should say MFD (thats the touchscreen) or navigation.
> If say Radio 2012 is missing I think for ICE cars (petrol and diesel) the BIC ECU has to be modified in a Script Box with a script number for your car , something like that , to 'wake it up' - my car where I rescued it is a Zoe EV and it does not have the BIC ecu but ICE Renaults do apparently. There was a guy who gave instruction about wiring modification for the BIC and what script number to put in to wake up the ECU (either Radio2012 or MFD) sorry its very confusing but I will see if I can find it.

Is a mediaNav 1

**Reply by morganclio** [reply]
Does it mean that I have the wrong ELM cable or I didn't switch the cables in the splitter correctly?
![20250329_073440](https://github.com/user-attachments/assets/a042a4d8-d760-4e12-9985-8347344abbb8)

**Reply by morganclio** [reply]
So now it looks like that but when I click WRITE ALL , it thinks but than everything goes to red and still not working
![20250329_120305](https://github.com/user-attachments/assets/a4ed5754-8e61-4b0a-ac64-e6626b81ca9d)
![20250329_121324](https://github.com/user-attachments/assets/e4035c3f-1fea-4279-a3dd-4a23b556ddab)

**Reply by AndyIreland** [reply]
ok , then press the refresh button in DDT4ALL and see if it has written the parameters right , and then if it has press the 'Hard Reset' button in DDT4ALL (not the soft reset button) and see what happens

**Reply by morganclio** [reply]
Hard reset in the yellow box 
Or under number 11?
![Uploading 20250329_122759.jpg…]()

**Reply by morganclio** [reply]
![20250329_122759](https://github.com/user-attachments/assets/197abb72-e2c1-4ce3-932e-1e3a62292914)
![20250329_122803](https://github.com/user-attachments/assets/d0545958-d5ac-46a4-a12a-6e84e6d72b88)

**Reply by AndyIreland** [reply]
> Does it mean that I have the wrong ELM cable or I didn't switch the cables in the splitter correctly? ![20250329_073440](https://private-user-images.githubusercontent.com/204682888/428277059-a042a4d8-d760-4e12-9985-8347344abbb8.jpg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDMyNTE2MzYsIm5iZiI6MTc0MzI1MTMzNiwicGF0aCI6Ii8yMDQ2ODI4ODgvNDI4Mjc3MDU5LWEwNDJhNGQ4LWQ3NjAtNGUxMi05OTg1LTgzNDczNDRhYmJiOC5qcGc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUwMzI5JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MDMyOVQxMjI4NTZaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT00YjliNjI4YjU4ZDhiMDFlNTU2YjJjMDhiYmZhNWFhY2E2ZmM1MzQ0OTJhZjYxZDI3ZDhhYmM3OWJkNjgwMzU0JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.nJx3Snm7EFbQ_wjfkTd7d-5BzfqVeCynf7qPYGFtXhQ)


Check your adaptor has a PIC18F25K80 chip in it -  sure fire way split it open and with a magnifier glass look for PIC18F25K80 stamped onto microchip. 

check your fly cable is wired up properly and set CAN  timeout bottom of DDT4ALL screen to 600ms

**Comment by AndyIreland** [comment]
this first image in this post:
![image](https://github.com/user-attachments/assets/8c9691b0-5157-4c78-a215-06a47c750b30)

is a Rlink1 TomTom and not a medianav. - medianav have the USB socket to right of screen and has no SD card Map reader in the car

**Reply by Furtif** [reply]
Ok but mediaNav 1 as same profil Xd (captur 1 clio 1 etc)

**Reply by Furtif** [reply]
But in all stuff boy have format ecu need reconf..

**Reply by morganclio** [reply]
> format

Yes , I do believe this happened. The radio was cleared and now it needs to be reconfigured...

**Comment by morganclio** [comment]
Should I press "Reset Hard" in the 11-ECU Reset?

**Reply by AndyIreland** [reply]
all the hard reset buttons do the same thing as pressing the home button on the rlink 1 screen 5 times I think

**Comment by morganclio** [comment]
It didn't work out 😔

**Reply by AndyIreland** [reply]
thats a shame , would you like me to try log on remotely to see if there is anything I can see that maybe wrong?

you need 
windows laptop

**Comment by morganclio** [comment]
Maybe my cable is wrong so it can read but can't write... I don't want to
open and see what's inside as I will return it and order different one on
Amazon that it says it has this chip. Than if this will not work we might
try the remote option like you said if you don't mind

**Reply by AndyIreland** [reply]
i think if it can read then it will be ok it must be compatible . 
Did you fill out the Radio2012  ecu parameters as well or just the MFD

**Reply by morganclio** [reply]
I fill the numbers 12,13 and 14  from the MFD .
Radio2012 🤔 where is that ? This definitely I didn't touch it

**Reply by AndyIreland** [reply]
under 'audio unit':

Dont forget to put your VIN and CRC info under that too

![image](https://github.com/user-attachments/assets/753d8553-2957-452b-956f-e1c39d43a381)

**Reply by AndyIreland** [reply]
in car with rlink1 you have 2 different units the Radio part and then the MFD (multifunction device touch screen)  - radio unit normally hidden behind dash somewhere

**Comment by morganclio** [comment]
So from this screen after I click refresh
![20250329_125103](https://github.com/user-attachments/assets/47c9a33b-b46d-4dfc-9248-18f627a8ccb4)



 I'm getting this screen
![20250329_125122](https://github.com/user-attachments/assets/c76ab82c-2ac6-4913-ba4f-2c422a445075)



So it looks like it's not writing the data of I'm correct?

**Comment by morganclio** [comment]
I will give it a go in few minutes...

**Comment by morganclio** [comment]
My friend I ow you big !!!! The Radio 2012 was the problem! Got the radio
on and the RS MODE 😁😁😁😁

**Reply by AndyIreland** [reply]
Oh brilliant news I am so glad you got it sorted in the end - fantastic job

**Comment by morganclio** [comment]
Just I don't have audio 🤣🤣 🤦‍♂️ everything works just no sound coming
from speakers

**Reply by AndyIreland** [reply]
double check this in radio2012:

![image](https://github.com/user-attachments/assets/6050e9d7-02a6-4c4b-a399-f02d373bed99)

**Comment by morganclio** [comment]
Yes it is like that. When I press ECU Hard Reset you can hear in speakers
some kind of sound like the power is off/on but that's it

**Comment by morganclio** [comment]
It's all working now! Got the sound. Was checking all settings,my external
amp, Fuses etc etc. You know what fixed the problem? I just lock the car,
gone home for maybe 10min and come back as wanted to strip glove box and
check the connections. Start the car and bum sound from the R effect Nissan
GTR was blowing from speakers 🔇
Thank you very much for all your help. Much appreciated

**Reply by AndyIreland** [reply]
excellent - yes that cures some things sometimes, walking away from car and after a while it all shuts down properly to save running down the 12v battery and sometimes resets things
